### ISSUE TITLE
Various issues when used with Material Kit / Bootstrap 3 from CDN, BS4

### ISSUE THREAD

**Issue by 4levels** [creativetimofficial/material-bootstrap-wizard#25]
Hi @timcreative,

very impressive project you've made, I'm very enthousiastic about Material Kit and this bootstrap wizard enhancement!

I did however struggle a lot to get this to play nice with a project I'm working on, and this seems to be related to some code rot in this repo.
- the data-color attribute only seems to change the positional slider color, but not the colors of the button classes
- when using it with your material-kit, things look even worse: positioning of the append-icons doesn't work, font sizes become way to small, input-groups are not being stretched to the col-* width's
- the image-container class makes everything dark when not setting a background-image which is really hard to detect
- there's no package.json in this repo to change the color scheme in this library (this worked perfectly in material-kit) so there seems no way to adjust the color scheme at all
- using it with BS4 doesn't work either: allthough the positioning issue of the input addon's disappeared, other issues appeared, similar to the issues with using your material kit separately

TL;DR;
Is it possible to override the color schemes?  I'm really disliking the pink as primary color!

Thanks again!

Erik

**Comment by dragosct** [comment]
Hi, @4levels! Thank you for using our product and sorry for the late response. Please, tell me did you use the Material Kit on BS3 and you tried to import this wizard to your project? Regards, Dragos
